### DISCUSSION TITLE
When the slider is vertical the Minimum value is up and the Maximum value is down. Is there a way to make it the opposite?

### DISCUSSION THREAD

**Discussion by basilbai** [react-native-elements/react-native-elements#3338] [Q&A]
When the slider is vertical the Minimum value is up and the Maximum value is down. Is there a way to make it the opposite?

**Comment by sumj25** [comment]
I'm working on this issue. I would like to claim it.

**Comment by sumj25** [comment]
Hii @basilbai I hope you are in the best of your health. You can change {vertValue} to {maximumValue(50)-(vertValue-minimumValue(20))} and it will gives Maximum value is up and the minimum value down.

**Reply by ult0** [reply]
This does not fix it for me. It just makes the slider glitch out and place the thumb marker on the opposite side of the slider
